GOVERNANCE.md

```diff
@@ -0,0 +1,106 @@
+# Project Governance
+
+- [Maintainers](#maintainers)
+- [Becoming a Maintainer](#becoming-a-maintainer)
+- [Meetings](#meetings)
+- [CNCF Resources](#cncf-resources)
+- [Code of Conduct Enforcement](#code-of-conduct)
+- [Voting](#voting)
+
+## Maintainers
+
+KubeVirt Maintainers govern the project. Maintainers collectively manage the 
+project's resources and contributors, and speak for the project in public.  The
+maintainers collectively decide any questions that cannot be resolved at the 
+individual repository level, and provide strategic guidance for the project
+overall.
+
+The current maintainers can be found in [MAINTAINERS](./MAINTAINERS.md).  
```

Comment:
Shouldn't this PR include this file, even if empty? Though I suspect we'd better initialize it with the list of people who actually do this work these days.

Instead of a new file, how about adding a `maintainers` alias into the existing OWNERS file? The function of these files would be very similar, and sure to confuse me.

Reply:
Dan: if you read this governance proposal, it specifically separates OWNERS for Maintainers.  That's really the meat of the proposal.  So adding the separate file instead.